aten/src/ATen/native/quantized/cpu/qadd.cpp

```diff
@@ -9,48 +9,66 @@
 namespace at {
 namespace native {
 namespace {
+
+inline void check_inputs(const Tensor& qa, const Tensor& qb) {
+  TORCH_CHECK(qa.qscheme() == kPerTensorAffine ||
+              qa.qscheme() == kPerTensorSymmetric,
+              "Only per tensor quantization is suuported in Add.");
+  TORCH_CHECK(qa.qscheme() == qb.qscheme(),
```

Comment:
We can mix and match qschemes for a and b, so we should only check:
```
TORCH_CHECK(qa.qscheme() == kPerTensorAffine ||
              qa.qscheme() == kPerTensorSymmetric,
              "Only per tensor quantization is supported in Add.");
TORCH_CHECK(qb.qscheme() == kPerTensorAffine ||
              qb.qscheme() == kPerTensorSymmetric,
              "Only per tensor quantization is supported in Add.");
```

Reply:
Will send it as a separate diff

Reply:
Created an issue to keep track of this task: #24215